.changeset/cuddly-islands-cross.md

```diff
@@ -0,0 +1,7 @@
+---
+"example-persisted-operations": minor
+"@graphql-mesh/config": minor
```

Comment:
These packages are v0 so let's keep them bumping as patch, because minor means breaking for v0